datastore/writer/postgresql_backend/sql_occ_locker_backend_service.py

```diff
@@ -92,17 +92,14 @@ def get_locked_fqfields(self, fqfields: Dict[str, int]) -> List[str]:
         ]
         query = dedent(
             f"""\
-            with all_together as (
-                select e.fqid, cf.collectionfield from events e
-                    inner join events_to_collectionfields ecf on e.id=ecf.event_id
-                    inner join collectionfields cf on ecf.collectionfield_id=cf.id
-                where {event_filter})
-            select fqid || %s || split_part(collectionfield, %s, 2) from all_together where {collectionfield_filter}"""
+            select e.fqid || %s || split_part(cf.collectionfield, %s, 2) from events e
```

Comment:
Do you have time comparisons for turning back to the non CTE-version. I remember that CTE reduces the time a lot, see https://github.com/OpenSlides/openslides-datastore-service/pull/203

Reply:
I indeed forgot that this was already changed some time ago. On may machine, the CTE version takes 2:02min for an example query issued during a meeting import, while the new query takes about 8sec. It seems that CTE is quicker than using subqueries, but an "all-in-one" query still beats it by a landslide.